README.md

```diff
@@ -48,6 +48,19 @@ pointing to invalid nightlies after a failed build, we publish a simple
 nightly version file to the CDN too.
 
 
+## Dependency Cycle: natscli
+
+Note that the natscli repo's `.goreleaser.yml` file now depends upon this site
+for all builds, to fetch the zsh completion.
+(See <https://github.com/nats-io/natscli/pull/1227>)
+
+If moving or turning down this repo, address that dependency.
+
+In practice, as long as the site remains up, there is no "cycle": our nightly
+builds depend upon the natscli GoReleaser configuration, which depends upon
+our static site.
```

Comment:
Is this really true? I did not realise that adding `generate_completions_from_executable` would make a dependency on this - that's a home-brew feature.

If that is true, then I vote for reverting that PR instead.

I am not in favour of this repo existing, let alone depending on it.

Reply:
Ah, I missed that the logic which added the dependency was removed in the next commit and I apparently neglected to check HEAD, sorry.

```
* |   e834c6cc Merge pull request #1227 from GaMoCh/add-man-page-and-zsh-completion
|\ \  
| * | fd30578f (pr/1227) Remove man page
| * | 72cb36da Use generate_completions_from_executable
| * | 36f3fec3 Add man page and zsh completion
|/ /  
* |   d885bc61 Merge pull request #1226 from ripienaar/audit_markdown
```

Commit `36f3fec3` added the dependency upon this repo but commit `72cb36da` removed it.